### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by nim65s**
- update robotpkg & apt instructions
- use an absolute http link for non versionned files
- cherry-pick "corrections found during class"

### CODE REVIEW

File: tp1_directgeom/main.asciidoc [hunk 1]
```diff
@@ -136,9 +136,9 @@ pkg  = '/home/student/models/'
 urdf = pkg + '/ur_description/urdf/ur5_gripper.urdf'
 robot = RobotWrapper(urdf,[pkg,])
 ----
-The code of the RobotWrapper class is in /path/to/pinocchio//src/python/romeo_wrapper.py . Do not hesitate to have a look at it and to take inspiration from the implementation of the class functions.
+The code of the RobotWrapper class is in /opt/openrobots/lib/python2.7/site-packages/pinocchio/robot_wrapper.py . Do not hesitate to have a look at it and to take inspiration from the implementation of the class functions.
```
Comment (nmansard) [inline tp1_directgeom/main.asciidoc:139]: We should not over-specified the tuto toward robotpkg installation.
I suggest: the code of the RobotWrapper class is in the Pinocchio source package (in pinocchio/bindings/python/script/robot_wrapper.py) and is installed when compiling the package in lib/python2.7/site-packages/pinocchio/robot_wrapper.py (when installing using robotpkg or inside the virtual-box, the exact path is /opt/openrobots/lib/python2.7/site-packages/pinocchio/robot_wrapper.py ).

File: tp0_install/install.sh [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #! /bin/sh
 
-### Install basic dependencies 
-sudo apt-get install -y cmake cmake-curses-gui cmake-qt-gui curl doxygen g++ ipython libassimp-dev assimp-utils libboost-dev omniidl-python omniorb omniorb-idl omniorb-nameserver libomniorb4-dev libomnithread3-dev python python-matplotlib python-numpy python-scipy python2.7 qgit libbz2-dev zlib1g-dev libssl-dev pax tar libeigen3-dev libtinyxml-dev liburdfdom-dev libboost-all-dev libpcre3-dev libopenscenegraph-dev libqt4-dev python-qt4-dev libncurses-dev python-pip cython python-h5py
+### Setup environment
```
Comment (nmansard) [inline tp0_install/install.sh:3]: 1. we should keep the list of dependencies, should we not. Even if not directly useful, it seems to be an important part of the documentation.
2. Are you sure that none of these packets are useful. For example, h5py is needed for running the install of tensor-flow, is it not?

File: tp1_directgeom/main.asciidoc (hunk 1 above)
Comment (olivier-stasse) [reply tp1_directgeom/main.asciidoc:139]: if the $PYTHONPATH is correctly set, references to robotpkg should not be needed.

File: tp1_directgeom/main.asciidoc [hunk 3]
```diff
@@ -136,11 +136,11 @@ pkg  = '/home/student/models/'
...
+UR5 is a fixed robot with one 6-DOF arms developed by the Danish company Universal Robot. All its 6 joints are revolute joints. Its configuration is in R^6 and is not subject to any constraint. The model of UR5 is described in a URDF file, with the visuals of the bodies of the robot being described as meshed (i.e. polygon soups) using the Collada format ".dae". Both the URDF and the DAE files are available in the attached ZIP archive. Uncompressed it in the VirtualBox, for example in the directory "/home/student/models".
 
-* UR5 model and code link:ur5.zip[*ur5.zip*]
+* UR5 model and code link:http://homepages.laas.fr/nmansard/teach/supaero2017/ur5.zip[*ur5.zip*]
```
Comment (nmansard) [inline tp1_directgeom/main.asciidoc:143]: Good instantaneous patch, but seems dangerous in the long term. Could we keep both?

UR5 model and code link:ur5.zip[*ur5.zip*] (link should be automatically setup, see hard-code link of previous year class here: http://homepages.laas.fr/nmansard/teach/supaero2017/ur5.zip[*ur5.zip*])

**Comment by CamilleMaurice** [comment]
About the update of Nov17, "Update installation instructions". I did not manage to make osg-dae work on Ubuntu 16.04. The command osgconv cannot load the .dae files on my machine.

**Comment by jcarpent** [comment]
It seems that osg-dae plugin is only working with 14.04 and 12.04. 
@jmirabel Do you know if the plug-in is working on 16.04?
@nim65s If yes, can you push it on robotpkg?

**Comment by jcarpent** [comment]
@CamilleMaurice Thank you for your feedback.

**Comment by jmirabel** [comment]
It is working on 16.04. With robotpkg, the plugin is installed in `lib64` and not `lib`. Can you check your `LD_LIBRARY_PATH` (or post its value here) ?

**Pull request merged**